man/idig_search_records.Rd

```diff
@@ -57,6 +57,30 @@ this:
 \preformatted{
 rq <- list("scientificname"=list("type"="exists"), "family"="asteraceae")
 }
+
+An example of a more complex JSON query
+
+{```
+
+  "geopoint": {
+    "type": "geo_bounding_box",
+    "top_left": {
+      "lat": 19.23,
+      "lon": -130
+    },
+    "bottom_right": {
+      "lat": -45.1119,
+      "lon": 179.99999
+    }
+  }
+}
+```
+To rewrite this in R for use as the rq parameter to
+\code{idig_search_records} or \code{idig_search_media}, it would look like
+this:
+\preformatted{
+rq <- list(geopoint=list(type="geo_bounding_box", top_left=list(lat=19.23, lon=-130), bottom_right= list(lat=-45.1119, lon= 179.99999)))
```

Comment:
insert a couple of line returns here, so the line is about 80 character wide. Otherwise, part of the code will be hidden in the PDF file of the help.